gnucash/gnome/gnc-budget-view.c

```diff
@@ -1230,7 +1230,12 @@ gbv_refresh_col_titles(GncBudgetView *view)
     gchar title[MAX_DATE_LENGTH];
     GList *col_list;
     gint i;
+    gboolean editable, read_only;
+    GtkCellRenderer *renderer;
+    GDate *readonly_threshold =
+        qof_book_get_autoreadonly_gdate(gnc_get_current_book());
```

Comment:
Do you really want to apply the read-only threshold to budgets? It's intended to apply only to transactions so that you can't create transactions more than n days in the past nor modify transactions more than n days old. Budgets aren't part of the accounting record and it would seem that users might want retrospective budget changes even when they don't want to allow transaction changes.

Reply:
Will leave this for another PR